src/engine/ExportQueryExecutionTrees.cpp

```diff
@@ -400,7 +405,9 @@ ExportQueryExecutionTrees::selectQueryResultToCsvTsvOrBinary(
 
   static constexpr char separator = format == MediaType::tsv ? '\t' : ',';
   // Print header line
-  auto variables = selectClause.getSelectedVariablesAsStrings();
+  std::vector<std::string> variables =
+      selectClause.getSelectedVariablesAsStrings();
+  variables.push_back(string("?completedWord"));
```

Comment:
I see, this currently ALWAYS adds the word variable and always with a fixed name.
This variable should not be automatically added, but should be handled similarly to the `textscore` variables. 
You can have a look at how these are handled within the Sparql parser to make tthe queries work, but we can also talk about this tomorrow.